<?xml version="1.0" encoding="UTF-8"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1482057538755" stop="1482057545096">
	<name>Login to bank application</name>
	<title>Login to bank application</title>
	<test-cases>
		<test-case start="1482057538760" stop="1482057543837" status="passed">
			<name>Validate user credentials</name>
			<title>Validate user credentials</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="1dddd3f5-1b4b-4385-aea7-80e708523b5e-attachment.png" type="image/png" size="45617"/>
			</attachments>
		</test-case>
		<test-case start="1482057543838" stop="1482057545095" status="failed">
			<name>Add New Customer</name>
			<title>Add New Customer</title>
			<description></description>
			<labels>
			</labels>
			<parameters>
			</parameters>
			<steps>
			</steps>
			<attachments>
				<attachment title="Screenshot" source="09823d5d-c7a7-4292-bccd-841ec3b11467-attachment.png" type="image/png" size="26640"/>
			</attachments>
			<failure>
				<message>Failed: No element found using locator: by.cssContainingText(&quot;a[href*=&apos;addcustomerpage&apos;]&quot;, &quot;undefined&quot;)</message>
				<stack-trace>NoSuchElementError: No element found using locator: by.cssContainingText(&quot;a[href*=&apos;addcustomerpage&apos;]&quot;, &quot;undefined&quot;)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\built\element.js:807:27
    at ManagedPromise.invokeCallback_ (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:1379:14)
    at TaskQueue.execute_ (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:2913:14)
    at TaskQueue.executeNext_ (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:2896:21)
    at asyncRun (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:2775:27)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:639:7
    at process._tickCallback (internal/process/next_tick.js:103:7)Error
    at ElementArrayFinder.applyAction_ (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\built\element.js:460:27)
    at ElementArrayFinder._this.(anonymous function) [as click] (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\built\element.js:104:30)
    at ElementFinder.(anonymous function) [as click] (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\built\element.js:828:22)
    at Object.&lt;anonymous&gt; (C:\Users\Dipjyoti\WebstormProjects\quickstart\e2e\PageObjects\BankDashboard.js:29:29)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasminewd2\index.js:94:23
    at new ManagedPromise (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:1082:7)
    at controlFlowExecute (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasminewd2\index.js:80:18)
    at TaskQueue.execute_ (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:2913:14)
    at TaskQueue.executeNext_ (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:2896:21)
    at asyncRun (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:2820:25)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:639:7
    at process._tickCallback (internal/process/next_tick.js:103:7)
From: Task: Run it(&quot;Add New Customer&quot;) in control flow
    at Object.&lt;anonymous&gt; (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasminewd2\index.js:79:14)
    at attemptAsync (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1916:24)
    at QueueRunner.run (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1871:9)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1898:16
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:1842:9
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasminewd2\index.js:16:5
    at ManagedPromise.invokeCallback_ (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:1379:14)
    at TaskQueue.execute_ (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:2913:14)
    at TaskQueue.executeNext_ (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:2896:21)
    at asyncRun (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:2775:27)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\selenium-webdriver\lib\promise.js:639:7
    at process._tickCallback (internal/process/next_tick.js:103:7)
From asynchronous test: 
Error
    at Suite.&lt;anonymous&gt; (C:\Users\Dipjyoti\WebstormProjects\quickstart\e2e\PageObjects\BankDashboard.js:28:3)
    at addSpecsToSuite (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:833:25)
    at Env.describe (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:802:7)
    at describe (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\jasmine-core\lib\jasmine-core\jasmine.js:3375:18)
    at Object.&lt;anonymous&gt; (C:\Users\Dipjyoti\WebstormProjects\quickstart\e2e\PageObjects\BankDashboard.js:8:1)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\coffee-script\lib\coffee-script\register.js:45:36)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\node_modules\jasmine\lib\jasmine.js:71:5
    at Array.forEach (native)
    at Jasmine.loadSpecs (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\node_modules\jasmine\lib\jasmine.js:70:18)
    at Jasmine.execute (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\node_modules\jasmine\lib\jasmine.js:154:8)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\built\frameworks\jasmine.js:129:15
    at Function.promise (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:682:9)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\protractor\built\frameworks\jasmine.js:101:14
    at _fulfilled (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:796:13)
    at C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:556:49
    at runSingle (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:137:13)
    at flush (C:\Users\Dipjyoti\WebstormProjects\quickstart\node_modules\q\q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)</stack-trace>
			</failure>
		</test-case>
	</test-cases>
</ns2:test-suite>